src/packages/src/xroad/ubuntu/generic/xroad-signer.postinst

```diff
@@ -41,6 +41,18 @@ if [ "$1" = configure ]; then
         crudini --del ${local_ini} common default-signature-algorithm
     fi
 
+    # migrate keys to a new directory
+    signer_folder=/etc/xroad/signer
+    if [[ ! -d ${signer_folder}/softtoken ]]
+    then
+        mkdir -p -m 0750 ${signer_folder}/softtoken.tmp
+        test -f ${signer_folder}/.softtoken.p12 && cp -a ${signer_folder}/.softtoken.p12 ${signer_folder}/softtoken.tmp/.softtoken.p12
+        cp -a ${signer_folder}/*.p12 ${signer_folder}/softtoken.tmp/ || :
```

Comment:
Removed the ` || :` part which would suppress the failure. Also added a test to check if any `*.p12` files exist before trying to copy them –  script would otherwise fail if keys were missing.